[nightly] Javascript tests pipeline is failing

Build: https://drone.owncloud.com/owncloud/calendar/1983/8/4

```bash
+ make test-js
yarn install
yarn install v1.22.22
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
error Error: https://codeload.github.com/kayellpeee/hsl_rgb_converter/tar.gz/3936a3896dd2f45a5c66317aab6f9175c9fde9fb: Request failed "404 Not Found"
    at ResponseError.ExtendableBuiltin (/usr/lib/node_modules/yarn/lib/cli.js:696:66)
    at new ResponseError (/usr/lib/node_modules/yarn/lib/cli.js:802:124)
    at Request.<anonymous> (/usr/lib/node_modules/yarn/lib/cli.js:66750:16)
    at Request.emit (events.js:400:28)
    at Request.module.exports.Request.onRequestResponse (/usr/lib/node_modules/yarn/lib/cli.js:142287:10)
    at ClientRequest.emit (events.js:400:28)
    at HTTPParser.parserOnIncomingClient (_http_client.js:647:27)
    at HTTPParser.parserOnHeadersComplete (_http_common.js:127:17)
    at TLSSocket.socketOnData (_http_client.js:515:22)
    at TLSSocket.emit (events.js:400:28)
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
make: *** [Makefile:175: node_modules] Error 1
```

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "angular-ui-bootstrap": "^2.5.0",
     "autosize": "^5.0.2",
     "fullcalendar": "^3.10.5",
-    "hsl_rgb_converter": "kayellpeee/hsl_rgb_converter",
+    "hsl_rgb_converter": "o-klp/hsl_rgb_converter",
     "ical.js": "DeepDiver1975/ical.js#timezone-fix-and-build",
     "jquery-timepicker": "fgelinas/timepicker#883bb2cd94",
     "jstzdetect": "pellepim/jstimezonedetect"

